packages/web-app-files/src/components/Collaborators/Collaborator.vue

```diff
@@ -140,8 +141,8 @@
         <div class="uk-flex uk-flex-nowrap uk-flex-middle">
           <oc-button
             v-if="$_editButtonVisible"
-            :aria-label="$gettext('Edit share')"
-            :uk-tooltip="$gettext('Edit share')"
+            :aria-label="$gettext(`Edit share with ${ collaborator.collaborator.displayName }`)"
```

Comment:
Unfortunately doesn't work like this. For inserting a variable into a translation, you need to use `gettextInterpolate`. And that's a 2-step-process. Translating the string (still with the variable) and then injecting the value into the translated string. See docs: https://github.com/Polyconseil/vue-gettext#interpolation-support-1 - I'd recommend to do this in a computed property.

Reply:
Yeah I recall seeing this somewhere in the codebase, will push an update later today! Thanks ;)

Reply:
👍